velox/docs/functions/spark/datetime.rst

```diff
@@ -61,6 +61,10 @@ These functions support TIMESTAMP and DATE input types.
     ``day`` need to be from 1 to 31, and matches the number of days in each month.
     days of ``year-month-day - 1970-01-01`` need to be in the range of INTEGER type.
 
+.. spark:function:: months_between(timestamp, timestamp, boolean) -> double
```

Comment:
Please distinguish the parameters and use the their names in the content.

Reply:
done